lib/ingram_micro/elements/shipment_status.rb

```diff
@@ -1,44 +1,47 @@
-class IngramMicro::ShipmentStatus < IngramMicro::BaseElement
+module IngramMicro
+  class ShipmentStatus < BaseElement
+    DEFAULTS = {
+      customer_id: nil,
+      business_name: nil,
+      detail: nil,
+      line_items: []
+    }.freeze
 
-  DEFAULTS = {
-    customer_id: nil,
-    business_name: nil,
-    detail: nil,
-    line_items: []
-  }
-
-  def defaults
-    DEFAULTS
-  end
+    def defaults
+      DEFAULTS
+    end
 
-  def initialize(options={})
-    super
-    @element[:order_header] ||= IngramMicro::SalesOrderHeader.new
-    check_line_items
-    @element[:detail] ||= IngramMicro::Detail.new({line_items: @element[:line_items]})
-  end
+    def initialize(options = {})
+      super
+      @element[:order_header] ||= SalesOrderHeader.new
+      check_line_items
+      @element[:detail] ||= Detail.new(line_items: @element[:line_items])
+    end
 
-  def build(builder)
-    builder.send('header') do
-      builder.send('customer-information') do
-        builder.send 'customer-id', @element[:customer_id]
-        builder.send 'business-name', @element[:business_name]
+    def build(builder)
+      builder.send('header') do
+        builder.send('customer-information') do
+          builder.send 'customer-id', IngramMicro.configuration.customer_id
+          builder.send 'business-name', @element[:business_name]
+        end
+      end
+      builder.send('detail') do
+        @element[:detail].build(builder)
       end
     end
-    builder.send('detail') do
-      @element[:detail].build(builder)
-    end
-  end
 
-  def check_line_items
-    if @element[:line_items].empty?
-      ss_line_item = IngramMicro::ShipmentStatusLineItem.new
-      @element[:line_items] << ss_line_item
+    def check_line_items
```

Comment:
I had just rewritten this to appease rubocop, but now that I look more closely I don't think this method offers desired behavior. It's adding an empty line item if there are none, whereas it should raise an error.